### PR TITLE
Add to readme and index how to start a new PyCon

### PR DESCRIPTION
**Pull request by lorenanicole**
# Scope

This pull request includes:
- `README.md` add how to start your own PyCon
- `index.html` add how to start your own PyCon

## Local Build Preview

image

### CODE REVIEW

**Review by ewdurbin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -48,6 +48,8 @@ <h1>Join us at PyCon</h1>
           <p>Whether you are an experenced programmer, a hobby hacker or an absolute beginner, we'd love to welcome you to the Python community.</p>
           <p>PyCons are hosted all around the world, mostly organized by volunteers from local Python communities.</p>
           <p>This means that each PyCon is unique, although all PyCons share the same spirit &mdash; to spread knowledge, learn from each other and make lasting connections.</p>
+          <p>If you are looking to start a new PyCon, you'll need to contact the <a href="https://www.python.org/psf/committees/#trademarks-work-group" style="color: white">Python Software Foundation trademark workgroup</a> for permission. Please confirm that: 1.) there is no other regional PyCon in your area (e.g. country) 2.) the PyCon has a Code of Conduct and 3.) the PyCon is a community conference and not for commercial purposes.
```
Comment (ewdurbin) [inline index.html]: I'd be curious how this would render with an actual `<ol>` instead of an inline, and difficult to parse, list.

Comment (lorenanicole) [reply index.html]: Ah yes, let me do that! 🤦

**Comment by lorenanicole** [comment]
Updated @ewdurbin here's the preview from local build: 

image

**Review by glasnt — APPROVED** [review]
🚢

**Pull request merged**